.circleci/config.yml

```diff
@@ -348,7 +350,7 @@ jobs:
           profile-name: default
       - run:
           name: build x86_64 release
-          command: cargo build --release -v
+          command: cargo build --release -v --features=uds-gateway
```

Comment:
Need to replicate this for `build_release_aarch64` & `build_release_osx`

Reply:
Done in 0ec0ec0c97362647768b980f547597cbdf84542b